api/channel.go

```diff
@@ -687,18 +673,11 @@ func removeChannelMember(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	} else {
 		channel := cresult.Data.(*model.Channel)
-		channelMember := cmcresult.Data.(model.ChannelMember)
```

Comment:
You stop using the resulting data from channel ```cmc``` but you still check for errors on it and it fetches from the database. If you're using this to check if the user is part of the channel then please use ```c.HasPermissionsToChannel()``` instead. You can search this file for examples on how to use it.